contracts/src/v0.8/dev/automation/2_1/Chainable.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.6;
+
+/**
+ * @title Chainable - the contract size limit nullifier
+ * @notice Chainable is designed to link together a "chain" of contracts through fallback functions
+ * and delegatecalls. All code is executed in the context of the head of the chain, the "master" contract.
+ */
+contract Chainable {
+  /**
+   * @dev addresses of the next contract in the chain **have to be immutable/constant** or the system won't work
+   */
+  address public immutable i_next;
+
+  /**
+   * @param next the address of the next contract in the chain
+   */
+  constructor(address next) {
+    i_next = next;
+  }
+
+  /**
+   * @notice returns the address of the next contract in the chain
+   */
+  function fallbackTo() external view returns (address) {
```

Comment:
just curious, where do you call this function?

Reply:
nowhere, it's just a public getter so that we can easily figure out the address of the next contract in the chain